deploy/helm/magda/charts/web-server/templates/configmap.yaml

```diff
@@ -7,5 +7,5 @@ data:
   web.json: '{
     "disableAuthenticationFeatures": {{ .Values.disableAuthenticationFeatures }},
     "baseUrl": {{ .Values.baseUrl | default "/" | quote }},
-    "showFallbackBanner": {{ .Values.showFallbackBanner }}
+    "fallbackUrl": {{ .Values.fallbackUrl }}
```

Comment:
Do we need double quote around `{{ .Values.fallbackUrl }}`.
i.e. `"{{ .Values.fallbackUrl }}"`
Asked because I got invalid `Invalid JSON config file: /etc/config/config.json` error
and the config map seems missing " around https://data.gov.au
```
web.json:
{ "disableAuthenticationFeatures": false, "baseUrl": "/", "fallbackUrl": https://data.gov.au }
```